3rd-assignment-modal/style.scss

```diff
@@ -0,0 +1,233 @@
+* {
+  box-sizing: border-box;
+}
+
+.col-4 {
+    width: 30%;
+}
+
+[class*="col-4"] {
+    float: left;
+    margin: 1.6%;
+    margin-bottom: 22px;
+    padding: 24px;
+    box-shadow: 0px 0px 11px grey;
+    border-radius: 15px;
+}
+
+body {
+    background-color: #fafafa;
+    margin: 0;
+}
+
+header {
+    background-color: white;
+    margin-bottom: 22px;
+    position: relative;
+    padding: 12px 29px;
+    box-shadow: 0px -2px 6px;
+    
+    i,img,input, button {
+        vertical-align: middle;
+    }
+
+    .search-box {
+        display: inline;
+    }
+
+    .logo-box {
+        display: inline;
+    }
+
+    i {
+        margin-right: 28px;
+    }
+
+    img {
+        margin-right: 120px;
+        height: 24px;
+        width: 80px;
+    }
+
+    input[list] {
+        font-size: 15px;
+        padding: 5px 10px;
+        width: 47%;
+    }
+    input[list]:focus {
+        box-shadow: inset 0px 0px 0.01em black;
+        border: 1px solid #5c5cd6;
+        outline: none;
+    }
+    button {
+        background-color: #fafafa;
+        margin-left: -4px;
+        text-align: center;
+        padding: 7px 18px;
```

Comment:
Check in whole scss and make sure that properties are alphabetically sorted

like  text-align should be after padding.